PROJECT

```diff
@@ -1,4 +1,7 @@
 domain: storageos.com
+layout:
+- go.kubebuilder.io/v3
+projectName: api-manager
```

Comment:
Doesn't adding `projectName` cause problems when generating new controllers?  Something to do with the resource naming...  I'll try.

Reply:
I don't think so. And I think our controller package directory structure is incompatible with what the controller-gen supports.

Reply:
It works well as you have it:

```
./bin/kubebuilder create api --group "" --version v1 --kind Volume
```
Creates:
```
GroupVersion = schema.GroupVersion{Group: "storageos.com", Version: "v1"}
```
:+1: